docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -21,22 +21,22 @@ You must have access to GitHub to install the {PROJECT_NAME} applications.
 [discrete]
 == Cloning the Git Repositories
 
-The following commands will clone the `master` branch of each repository and they collectively contain the latest tagged release of the {PROJECT_NAME} project.
-
+Clone the Git repositories as follows:
 [source,bash]
 ----
 git clone git@github.com:feedhenry-raincatcher/raincatcher-server.git
 git clone git@github.com:feedhenry-raincatcher/raincatcher-mobile.git
 git clone git@github.com:feedhenry-raincatcher/raincatcher-portal.git
 ----
 
-Previous versions are available through branches named with the `release-\{semver\}` convention.
-For example, in order to checkout version `0.0.1` of `raincatcher-server`, run the following commands after the cloning:
+The master branch of each Git repository provides the latest version. Previous
+versions have the `release-*` tag. For example, run the following command to
+checkout version `1.0.0`:
 
 [source,bash]
 ----
 cd raincatcher-server/
-git checkout release-0.0.1
+git checkout release-1.0.0
```

Comment:
💯  This was pointed as issue today to me. 
Good work!

Reply:
nice one. glad this is sorted!